core/cmd/local_client.go

```diff
@@ -436,12 +437,67 @@ func (cli *Client) VersionDatabase(c *clipkg.Context) error {
 		return fmt.Errorf("failed to initialize orm: %v", err)
 	}
 
-	version, err := migrations.Current(orm.DB)
+	db := postgres.UnwrapGormDB(orm.DB).DB
```

Comment:
Eventually I think we might want to instantiate it separately but this is fine for now